packages/fullstack/fullstack.txt

```diff
@@ -14,30 +18,40 @@ concat
 cond
 conda
 configurator
+contentful
 coworking
 datepicker
 deallocate
+deregister
 deserialize
 devkit
 directus
+dism
+dropdown
+envs
 eofline
 eprint
 eprintln
+esbenp
```

Comment:
What is the context for `exbenp`? Other than the person.

Reply:
Didn't saw it was the person '^^
Sry, i revert this commit.